Foundation/NSError.swift

```diff
@@ -64,7 +91,16 @@ public class NSError : NSObject, NSCopying, NSSecureCoding, NSCoding {
     }
     
     public func encodeWithCoder(aCoder: NSCoder) {
-        
+        if aCoder.allowsKeyedCoding {
+            aCoder.encodeObject(_domain.bridge(), forKey: "NSDomain")
```

Comment:
I know this would be an API change but I wonder if we could make an overload here to avoid needing to remember the .bridge(); not fully convinced that is a good idea but something to think about.